src/NuGet/Microsoft.Orleans.OrleansTelemetryConsumers.Counters.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Orleans.OrleansTelemetryConsumers.Counters</id>
+    <version>$version$</version>
+    <title>Microsoft Orleans Telemetry Consumer - Performance Counters</title>
+    <authors>Microsoft Research</authors>
+    <owners>Microsoft,Orleans</owners>
+    <projectUrl>https://github.com/dotnet/Orleans</projectUrl>
+    <licenseUrl>https://github.com/dotnet/Orleans#license</licenseUrl>
+    <iconUrl>https://raw.githubusercontent.com/dotnet/orleans/gh-pages/assets/logo_128.png</iconUrl>
+    <summary>
+      Microsoft Orleans Telemetry Consumer for Windows Performance Counters
+    </summary>
+    <description>
+      Windows Performance Counters implementation of Orleans Telemetry API.
+    </description>
+    <copyright>Copyright Microsoft 2015</copyright>
+    <tags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</tags>
+    <dependencies>
+      <dependency id="Microsoft.Orleans.Core" version="$version$" />
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="OrleansTelemetryConsumers.Counters.dll" target="lib\net451" />    
+    <file src="OrleansTelemetryConsumers.Counters.XML" target="lib\net451" />
+    <file src="OrleansTelemetryConsumers.Counters.pdb" target="lib\net451" />
+    <file src="$SRC_DIR$OrleansTelemetryConsumers.Counters\**\*.cs" target="src" />
+    <file src="OrleansCounterControl.exe" target="tools" />
+    <file src="OrleansCounterControl.exe.config" target="tools" />
+    <file src="Orleans.dll" target="tools" />
```

Comment:
I'm all for using the installer util, instead of maintaining our own.  It's just a question of resources.  If you want to make that part of this change, great, but if not we need the existing patterns to be preserved.